docs/architecture.md

```diff
@@ -6,27 +6,31 @@ This is how Style Dictionary works under the hood.
 
 Let's take a closer look into each of these steps.
 
-## 1. Find all token files
+## 1. Read the config
+
+Style Dictionary is a configuration based framework, you tell it what to do in a configuration file. Style Dictionary first reads this configuration to know what to do.
+
+## 2. Find all token files
 
 In your [config](config.md) file you define a `source`, which is an array of file paths. This tells Style Dictionary where to find your token files. You can have them anywhere and in any folder structure as long as you tell Style Dictionary where to find them.
 
-## 2. Deep merge token files
+## 3. Deep merge token files
 
 Style Dictionary takes all the files it found and performs a deep merge. This allows you to split your token files in any way you like, without worrying about accidentally overriding groups of tokens. This gives Style Dictionary a single, complete token object to work from.
 
-## 3. Split the platforms
+## 4. Iterate over the platforms
 
 For each platform defined in your [config](config.md), Style Dictionary will do a few steps to get it ready to be consumed on that platform. Everything that happens in a platform is non-destructive so you don't need to worry about one platform affecting another.
 
-## 4. Transform the tokens
+## 4a. Transform the tokens
 
 Style Dictionary now traverses over the whole token object and looks for design tokens. It does this by looking for anything with a `value` key. When it comes across a design token, it then performs all the [transforms](transforms.md) defined in your [config](config.md) in order.
 
-## 5. Resolve aliases / references to other values
+## 4b. Resolve aliases / references to other values
 
 After all the tokens have been transformed, it then does another pass over the token object looking for aliases, which look like `"{size.font.base.value}"`. When it finds these, it then replaces the reference with the transformed value. As we have a single complete token object, aliases can be in any token file and still work.
 
-## 6. Build files
+## 4c. Build files
```

Comment:
Mmm, not sure about 4c. Maybe internally it is, but I would imagine it to be a step on its own, from the perspective of the architecture. What do you think?